build.rs

```diff
@@ -1,14 +1,19 @@
-extern crate rustc_version;
-use rustc_version::{version, Version};
+extern crate version_check;
 
 fn main() {
-    if version().unwrap() >= Version::parse("1.25.0").unwrap() {
-        println!("cargo:rustc-cfg=rust_1_25");
-    }
-    if version().unwrap() >= Version::parse("1.26.0").unwrap() {
+    let version = match version_check::is_min_version("1.25.0") {
+        Some((true, version)) => version,
+        Some((false, _)) | None => return,
+    };
+    let mid = match version.split('.').nth(1).and_then(|s| s.parse::<u32>().ok()) {
```

Comment:
Wouldn't it be simpler to call `version_check::is_min_version` again?

Reply:
Unfortunately that currently reruns `rustc --version` which could make it slow :(

Reply:
On my system, it takes about 0.02 s to run `rustc --version` repeatedly, so I would expect that this is dwarfed by compilation time. Also, I think the previous version had three calls to `rustc --version` as well.